src/Outline.svg

```diff
@@ -1,3 +1,3 @@
 <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path fill-rule="evenodd" clip-rule="evenodd" d="M3 4C3 2.9 3.9 2 5 2H19C20.1 2 21 2.9 21 4V10.0179V10.9702V20C21 21.1 20.1 22 19 22H5C3.9 22 3 21.1 3 20V4ZM5 4V20H19V16.0004V9.99173V4H5ZM10.9958 11.0063V13.0063H13.9958V11.0063H10.9958ZM15 15V17H11V15H15ZM11 9H17V7H11V9ZM7 12.0063C7 11.4575 7.44905 11.0084 7.99789 11.0084C8.54674 11.0084 8.99579 11.4575 8.99579 12.0063C8.99579 12.5552 8.54674 13.0042 7.99789 13.0042C7.44905 13.0042 7 12.5552 7 12.0063ZM7.99789 15C7.44905 15 7 15.4491 7 15.9979C7 16.5467 7.44905 16.9958 7.99789 16.9958C8.54674 16.9958 8.99579 16.5467 8.99579 15.9979C8.99579 15.4491 8.54674 15 7.99789 15ZM7 7.99789C7 7.44905 7.44905 7 7.99789 7C8.54674 7 8.99579 7.44905 8.99579 7.99789C8.99579 8.54674 8.54674 8.99579 7.99789 8.99579C7.44905 8.99579 7 8.54674 7 7.99789Z" fill="black"/>
+<path d="M19 2C20.05 2 20.9178 2.82004 20.9945 3.85131L21 4V20C21 21.05 20.18 21.9178 19.1487 21.9945L19 22H5C3.95 22 3.08223 21.18 3.00551 20.1487L3 20V4C3 2.95 3.82004 2.08223 4.85131 2.00551L5 2H19ZM19 4H5V20H19V4ZM7.95197 15.0042C8.50082 15.0042 8.94987 15.4533 8.94987 16.0021C8.94987 16.5509 8.50082 17 7.95197 17C7.40313 17 6.95408 16.5509 6.95408 16.0021C6.95408 15.4533 7.40313 15.0042 7.95197 15.0042ZM15.944 15V17H9.94401V15H15.944ZM13.944 11V13H9.94401V11H13.944ZM7.95197 11C8.50082 11 8.94987 11.4491 8.94987 11.9979C8.94987 12.5467 8.50082 12.9958 7.95197 12.9958C7.40313 12.9958 6.95408 12.5467 6.95408 11.9979C6.95408 11.4491 7.40313 11 7.95197 11ZM16.944 7V9H9.94401V7H16.944ZM7.95197 7.00421C8.50082 7.00421 8.94987 7.45326 8.94987 8.00211C8.94987 8.55095 8.50082 9 7.95197 9C7.40313 9 6.95408 8.55095 6.95408 8.00211C6.95408 7.45326 7.40313 7.00421 7.95197 7.00421Z" fill="black"/>
 </svg>
```

Comment:
I know from looking in figma that it's supposed to be circular bullets with rectangular "text", but when the icon is viewed at 24x24 size, the bullet and rectangles blur together...
![CleanShot 2020-10-14 at 19 01 02](https://user-images.githubusercontent.com/545605/96057958-c083cb00-0e4f-11eb-8475-950c8b64b655.png)

Reply:
Fixed the point positions a little and added a pixel between the bullets and text lines.